pkg/lib/translation/service.go

```diff
@@ -119,17 +120,20 @@ func (s *Service) emailMessageHeader(name string, args interface{}) (sender, rep
 }
 
 func (s *Service) EmailMessageData(msg *MessageSpec, args interface{}) (*EmailMessageData, error) {
-	sender, replyTo, subject, err := s.emailMessageHeader(msg.Name, args)
+	data := map[string]interface{}{"ClientID": clientid.GetClientID(s.Context)}
```

Comment:
The translation service should handle constructing the message data only, it should not have context about client id. The client id should be provided by the caller. Thought?

Reply:
Any updates?

Reply:
@carmenlau If so then I think `ClientID` should be only included for magic link generate code email? Kind of weird to enforce client ID passing for messages that should not need client ID at all.

Reply:
Updated thanks!

Reply:
We need to have ClientID for all outgoing messages. I think EmailMessageData is the safest place to handle this. ClientID will never be missed. Please revert back to the original implementation.

Reply:
Updated :pray: